content/post/37-ipfs-conf-dev-meetings-and-user-registry.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2018-05-19
+url: 37-ipfs-conf-dev-meetings-and-user-registry.md
+title: Announcements: IPFS Conf,  IPFS & libp2p Developer Meetings and a User Registry
+author: flyingzumwalt
+---
+
+We’re very excited to announce two events that have been in the making for a very long time. In (tentatively) November we will host a public **IPFS Conf** in Lisbon. To prepare for that event, in July we will convene small **IPFS and libp2p Developer Meetings** in Berlin. Both of these events will be loosely inspired by PyCon and Docker Conf — events where community members play a very active role in shaping the agenda, and where a large portion of time is allocated to working meetings where Working Groups and product teams make the most of being together in-person. 
+
+# IPFSConf in November
```

Comment:
Non-essential, but in every other section it's written "IPFS Conf" (with a space).